### PR TITLE
Update content and video on 3.0 landing pages

### LINKED ISSUE #5116: Update video on 3.0 landing pages
A new InfluxDB 3.0 product overview video is now available and would be better to use on the serverless, dedicated, and clustered landing pages than the video on unlimited cardinality that is currently there.

Link to the 3.0 product overview: https://www.youtube.com/watch?v=uwqLWpmlQHM&list=PLYt2jfZorkDoJykM7NtpkqicLSSUfg5cH

##### Relevant URLs
- /influxdb/cloud-serverless/
- /influxdb/cloud-dedicated/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #5116 

- Updates the content on Serverless and Dedicated landing pages to use "InfluxDB 3.0" instead of "IOx."
- Replaces the unlimited cardinality video with a broader, more application introduction to InfluxDB 3.0 video.

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/_index.md [hunk 1]
```diff
@@ -21,14 +21,13 @@ time series use case.
 <a class="btn" href="{{< dedicated-link >}}">Request an InfluxDB Cloud Dedicated cluster</a>  
 <a class="btn" href="/influxdb/cloud-dedicated/get-started/">Get started with InfluxDB Cloud Dedicated</a>
 
-## The InfluxDB IOx storage engine
+## InfluxDB 3.0
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/_index.md:24]: "3.0" and not "v3"? Do we need to change that elsewhere?

File: content/influxdb/cloud-dedicated/_index.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ time series use case.
...
 
-**InfluxDB IOx** is InfluxDB's next generation storage engine that unlocks series
-limitations present in the Time Structured Merge Tree (TSM) storage engine.
-InfluxDB IOx allows infinite series cardinality without any impact on
-overall database performance. It also brings with it native
-**SQL support** and improved InfluxQL performance.
+**InfluxDB 3.0** is InfluxDB's next generation that unlocks series
+limitations present in the Time Structured Merge Tree (TSM) storage engine and
+allows infinite series cardinality without any impact on overall database performance.
+It also brings with it native **SQL support** and improved InfluxQL performance.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/_index.md:29]:
```suggestion
It also brings native **SQL support** and improved InfluxQL performance.
```

File: content/influxdb/cloud-serverless/_index.md [hunk 3]
```diff
@@ -21,37 +21,36 @@ date or through the Google Cloud Platform (GCP) or Azure marketplaces, see the
...
 
-**InfluxDB IOx** is InfluxDB's next generation storage engine that unlocks series
-limitations present in the Time Structured Merge Tree (TSM) storage engine.
-InfluxDB IOx allows infinite series cardinality without any impact on
-overall database performance. It also brings with it native
-**SQL support**<!-- and improved InfluxQL performance -->.
+**InfluxDB 3.0** is InfluxDB's next generation that unlocks series
+limitations present in the Time Structured Merge Tree (TSM) storage engine and
+allows infinite series cardinality without any impact on overall database performance.
+It also brings with it native **SQL support** and improved InfluxQL performance.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/_index.md:40]:
```suggestion
It also brings native **SQL support** and improved InfluxQL performance.
```

**Review by jstirnaman — APPROVED** [review]
Just some suggestions.

File: content/influxdb/cloud-dedicated/_index.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/_index.md:24]: There's still some debate on it. I figure we can come back and change it whenever things are finalized and the dust settles.

**Pull request merged**